rails.rb

```diff
@@ -3,11 +3,44 @@
 
 # 1. Create a method called hello_world that takes a language code (e.g. "es", "de", "ru", "ja") as an argument and returns the appropriate version of "Hello, World" for the given language. The default should be English. Test your code with THREE method calls. Expected output for hello_world 'es': 'Hola Mundo', hello_world 'de': 'Hallo Welt'
 
+def hello_world language
+    if language == "es"
+        puts "Hola, Mundo"
+    elsif language == "de"
+        puts "Hallo Welt"
+    elsif language == "ru"
+        puts "Privet, mir"
+    elsif language == "ja"
+        puts "Kon'nichiwa sekai"
+    else
+        puts "Hello, World"
+    end
+end
 
+hello_world "es"
+hello_world "de"
+hello_world "ru"
+hello_world "ja"
+hello_world "random"
 
 # 2a. Create a method called assign_grade that takes a number as an argument and returns the corresponding letter grade. Test your code with THREE method calls. Expected output for assign_grade 96: 'A', assign_grade 75: 'C'
+def assign_grade number
+  case number
+  when 90...100
+    "A"
+  when 80...90
+    "B"
+  when 70...80
+    "C"
+  when 60...70
+    "D"
+  when 0...60
+    "F"
+  end
+end
 
-
+number 78
+number 56
 
```

Comment:
Good use of the ... ! 
Remember to call the function name and return something to the console.
p assign_grade 78
p assign_grade 56